kubernetes/autosharding/service.yaml

```diff
@@ -1,21 +1,19 @@
 apiVersion: v1
 kind: Service
 metadata:
+  labels:
+    app.kubernetes.io/name: kube-state-metrics
+    app.kubernetes.io/version: v1.8.0-rc.0
```

Comment:
Should not all be `v1.8.0-rc.1`?

Reply:
It's whatever is in the `VERSION` file

Reply:
Not true, in the file above its v1.8.0-rc.1.